doc/source/whatsnew/v1.4.0.rst

```diff
@@ -401,6 +403,17 @@ Other API changes
 - :meth:`Index.get_indexer_for` no longer accepts keyword arguments (other than 'target'); in the past these would be silently ignored if the index was not unique (:issue:`42310`)
 - Change in the position of the ``min_rows`` argument in :meth:`DataFrame.to_string` due to change in the docstring (:issue:`44304`)
 - Reduction operations for :class:`DataFrame` or :class:`Series` now raising a ``ValueError`` when ``None`` is passed for ``skipna`` (:issue:`44178`)
+- Changed the ``name`` attribute of several holidays in
```

Comment:
These entries can be on the same line. This formatting is causing:

```
2021-11-24T23:39:37.6374111Z /home/runner/work/pandas/pandas/doc/source/whatsnew/v1.4.0.rst:424: WARNING: Unexpected indentation.
```

Reply:
I started that way, but it becomes a *really* long line.  Are nested tables a bad idea?

Reply:
Should be fixed (I believe; I ran the doc generation locally w/o apparent error)